src/main/java/com/treasuredata/client/model/TDColumn.java

```diff
@@ -41,8 +41,10 @@
 {
     static final byte[] LOG_TABLE_PUSHDOWN_KEY = "time".getBytes(UTF_8);
 
+    // The column SQL alias.
     private final String name;
     private final TDColumnType type;
+    // The actual column name.
```

Comment:
```suggestion
    // The physical column name in partition files
```

Reply:
A little bit confusing part is there are logical column names (alias) defined in TD API and physical column names used in MPC1 files. Actual is a relative adjective in this context.